packages/plugin-apex/src/commands/force/apex/test/run.ts

```diff
@@ -198,22 +210,25 @@ export default class Run extends SfdxCommand {
     }
 
     try {
-      if (result.summary.outcome === ApexTestRunResultStatus.Failed) {
+      if (
+        (result as TestResult).summary.outcome ===
```

Comment:
I'm getting `TypeError: Cannot read property 'outcome' of undefined` on this line if I don't specify `--resultformat`. For example `sfdx force:apex:test:run -n MyClass`.

Reply:
Nice catch, just addressed the issue